.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+version: 2.1
+orbs:
+  nais: navikt/nais-deployment@dev:add-command-to-login-to-github
+jobs:
+  build:
+    docker:
+      - image: circleci/golang:1.9.7
+    steps:
+      - checkout
+      - run: go get gopkg.in/mikefarah/yq.v2
+      - nais/docker-deploy:
+          image: navikt/di-dep-kafka-connect
+      - nais/login-to-github:
+          github-app-id: 29518
+      - run:
+          name: "Set image name in k8s config"
+          command: |
+            awk -v image="navikt/di-dep-kafka-connect:$CIRCLE_SHA1" '/image:/{c+=1}{if(c==2){sub("image: .*","image: " image,$0)};print}' nais.yaml > nais-tmp.yaml
+            mv nais-tmp.yaml > nais.yaml
+      - run:
+          name: "Create Github commit"
+          command: |
+            git config user.name didep-ci[bot]
+            git config user.email didep-ci[bot]@users.noreply.github.com
+
+            git add nais.yaml
+            git commit -m "[from ci] Update docker image in k8s config [ci skip]"
```

Comment:
I stedet for `git config user.name/user.email` så kan du velge å gjøre `git commit -m .. --author "didep-ci[bot] <didep-ci[bot]@users.noreply.github.com>"`

Men når jeg tenker meg om synes jeg egentlig det var mer lesbart sånn du allerede hadde gjort det :)